riak_test/common.erl

```diff
@@ -21,30 +21,42 @@
 
 -export([
   clean_clusters/1,
+  clean_clusters/2,
+  just_clean_clusters/1,
   setup_dc_manager/2]).
 
 %% It cleans the cluster and formes a new one with the same characteristic of the input one.
 %% In case the clean_cluster parameter is set to false in the riak_test configuration file
 %% the function simply returns the inputed set of clusters
 %% Clusters: A list of clusters as defined by rt
-clean_clusters(Clusters)->
-    Clean = rt_config:get(clean_cluster, true),
+clean_clusters(Clusters) ->
+    clean_clusters(Clusters, rt_config:get(clean_cluster, true)).
+
+clean_clusters(Clusters, Clean)->
     case Clean of
         true ->
 	    disconnect_dcs(Clusters),
             Sizes = lists:foldl(fun(Cluster, Acc) ->
-                                    rt:clean_cluster(Cluster),
-                                    Acc ++ [length(Cluster)]
+					rt:clean_cluster(Cluster),
+					Acc ++ [length(Cluster)]
                                 end, [], Clusters),
             Clusters1 = rt:build_clusters(Sizes),
             lists:foreach(fun(Cluster) ->
-                            rt:wait_until_ring_converged(Cluster)
+				  rt:wait_until_ring_converged(Cluster)
                           end, Clusters1),
             Clusters1;
         false ->
             Clusters
     end.
 
+just_clean_clusters(Clusters) ->
+    disconnect_dcs(Clusters),
+    lists:foldl(fun(Cluster, Acc) ->
+			rt:clean_cluster(Cluster),
+			Acc ++ [length(Cluster)]
+		end, [], Clusters),
+    ok.
+
```

Comment:
The lists:foldl should be simplified to:

lists:map(fun(Cluster) -> rt:clean_cluster(Cluster))

You never use the accumulated value.